ui/src/views/Provisioners/ViewWorkers/index.jsx

```diff
@@ -102,7 +117,9 @@ export default class ViewWorkers extends Component {
     } = this.props;
 
     this.setState({ filterBy: target.value });
-
+    target.value === 'Quarantined'
+      ? localStorage.setItem('quarantine', true)
+      : localStorage.clear();
```

Comment:
Instead of localStorage, let's rely on query parameters that way a user can send the link to another user and the deep linking will still work. We can do something similar to the pulse messages view.

https://taskcluster-ui.herokuapp.com/pulse-messages?bindings%5B0%5D%5Bexchange%5D=exchange%2Ftaskcluster-queue%2Fv1%2Ftask-pending&bindings%5B0%5D%5Bpattern%5D=%23 has `?bindings%5B0%5D%5Bexchange%5D=exchange%2Ftaskcluster-queue%2Fv1%2Ftask-pending&bindings%5B0%5D%5Bpattern%5D=%23`

Reply:
Oh I understand, users can't send links if we use localStorage. I will look into the implementation of pulse messages view.

Reply:
I added `this.props.history.replace` to the `handleFilterChange` function which gives the correct query string. But how can I retain the 'Quarantined' state on refresh. Please help.

![provi](https://user-images.githubusercontent.com/30138596/66705810-10194d80-ed49-11e9-9b64-3dc7a21c3d46.JPG)

Reply:
I think I managed to fix it. Can you take a look please?
@helfi92